docs/design-changes.md

```diff
@@ -0,0 +1,9 @@
+# Design Changes
+
+###### Design Change: Release of the react-native app
+The design of the react-native app will differ in a few points.
+* "Extras" will be changed to "Angebote"
```

Comment:
Do we use translations of "Angebote" in other languages or will it still be "Extras" there?